rust/Cargo.lock

```diff
@@ -5871,8 +5862,8 @@ checksum = "a2eb9349b6444b326872e140eb1cf5e7c522154d69e7a0ffb0fb81c06b37543f"
 
 [[package]]
 name = "str0m"
-version = "0.5.0"
-source = "git+https://github.com/firezone/str0m?branch=main#44b616945145033ade739f30cdce004728011619"
```

Comment:
This PR updates `str0m` in to include https://github.com/algesten/str0m/pull/515. Because `main` of str0m also includes https://github.com/algesten/str0m/pull/508 which caused problems when we integrated it into firezone, I've reverted that PR in our fork in https://github.com/firezone/str0m/commit/07c4fb0449db1fdde4b3421a61b666312ec81106.

Thus, our current diff to upsteam str0m looks like this: https://github.com/algesten/str0m/compare/main...firezone:str0m:main